src/js/utils/dom.js

```diff
@@ -160,7 +160,7 @@ export function createEl(tagName = 'div', properties = {}, attributes = {}, cont
     // method for it.
     } else if (propName === 'textContent') {
       textContent(el, val);
-    } else {
+    } else if (el[propName] !== val) {
```

Comment:
This function is also another big startup time user. We should make sure that the prop is not already equal to what we are going to set it to before doing it.